packages/react-keytips/package.json

```diff
@@ -1,9 +1,6 @@
 {
   "name": "@fluentui-contrib/react-keytips",
   "version": "0.0.1",
-  "dependencies": {
-    "@swc/helpers": "~0.5.2"
```

Comment:
what happened here ? without this dep it will explode in user land

Reply:
Good spot, thanks, will get it back